### ISSUE TITLE
损失函数的计算

### ISSUE THREAD

**Issue by ghost** [neukg/BiRTE#1]
你好，关于代码中损失函数的计算，我不太理解，主要是这一行，loss = relu(pred) - pred * targets + torch.log(1 + torch.exp(-1 * torch.abs(pred).to("cuda"))).to("cuda")，希望能得到您的回复
